### PR TITLE
Live Blogs

### PR DESCRIPTION
**Pull request by kand**
Add CMS live blog functionality.

**Updates**
1. Add `live-blog-entries` directive to list out the entries of a live blog.
2. `date-time-modal-opener` adds a clear button to clear out given date, as well as two additional attributes related to this new functionality:
  - `modal-on-clear` a callback when clear button is clicked
  - `modal-clear-text` text to display in clear button, defaults to `Clear`

### CODE REVIEW

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;iTudFdYD&#x2F;20-create-content-modal-entry-for-live-blog"> create content modal entry for live blog</a>

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;SJueuWQj&#x2F;21-entries"> Entries</a>

**Comment by kand** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Q957Q7EA&#x2F;8-responses-choose-from-and-create-personas-authors"> Responses: Choose from and create Personas&#x2F;Authors</a>

**Comment by kand** [comment]
@daytonn @collin @mparent61 @MelizzaP almost ready if you want to start taking a look

**Comment by kand** [comment]
test branch: http://liveblog.test.avclub.com/cms/app/edit/242371/

File: app/mocks/api-mock-data.js [hunk 1]
```diff
@@ -424,7 +424,9 @@ angular.module('bulbsCmsApp.mockApi.data', [])
         }],
         info_data: {
           data: {
-            entries: []
+            entries: [{
+              body: 'garbage<hr>garbage'
```
Comment (MelizzaP) [inline app/mocks/api-mock-data.js]: good use of garbage here

**Comment by MelizzaP** [comment]
lgtm so far

**Pull request merged**